commands/login.go

```diff
@@ -425,3 +416,224 @@ func IdentityString(pkt pktoken.PKToken) (string, error) {
 		return "Email, sub, issuer, audience: \n" + claims.Email + " " + claims.Subject + " " + claims.Issuer + " " + claims.Audience, nil
 	}
 }
+
+// Structure for that format :
```

Comment:
This should follow the godoc type format 

```
// ProviderConfig is our representation of the user supplied provider config string ...
```

Same idea for some of the other comments

https://go.dev/doc/comment